indigo_app/templates/indigo_api/work_form.html

```diff
@@ -164,6 +164,7 @@ <h5 class="card-header">Classification</h5>
 
       </div>
     </div>
+    {% block additional-information %}{% endblock %}
```

Comment:
This isn't necessary, because other templates can simply do this:

```
{% block work-form-details %}
  {{ block.super }}
  extra custom stuff
{% endblock %}
```

That's the reason for work-form-details being a block.